docs/alternative-environment-tips/docksal.md

```diff
@@ -0,0 +1,109 @@
+# Setting up BLT with Docksal
+
+By default, BLT is set to work with Drupal VM. It is easy to set up to use with Docksal, but
+you will have to install the Docksal `blt` addon command and make some configuration changes.
+
+## Setting Your Project to Use Docksal
+
+If you already have Docksal installed on your system, setting your BLT project to use Docksal only
+requires that you add a `.docksal` directory in your project and run `fin up` in the command prompt
+of your project directory. See the [Docksal Documentation](https://docs.docksal.io) for more specific
+Docksal information.
+
+## Using the Docksal `blt` Command
+
+BLT will install a `blt alias` command on your host system. Running the `blt` command will
+then use the resources of your host system. Instead, you need to make sure you're running
+the commands through the Docksal CLI service.
+
+Install the Docksal `blt` addon command:
+```
+fin addon install blt
+```
+
+Run any blt command with `fin` and you will be running the command within the Docksal CLI, e.g.:
+```
+fin blt doctor
+```
+
+
+## Setting the Database Connection and Drush Alias
+
+### New Project Setup
+
+The default Docksal database settings are different than the BLT defaults. If you have not already
+added special settings to the local.settings.php file that BLT creates, it is best to begin by
+following these steps:
+
+1. remove the `docroot/sites/default/settings/local.settings.php` and the `docroot/sites/default/local.drush.yml` files
+2. update the blt.yml file in your project's blt directory:
+
+    ```
+    project:
+      machine_name: myproject
+      local:
+        hostname: '${env.VIRTUAL_HOST}'
+    drupal:
+      db:
+        database: default
+        username: user
+        password: user
+        host: db
+        port: 3306
+    ```
+3. initiate and verify blt settings
+    ```
+    fin blt blt:init:settings
+    ```
+4. setup your Drupal site
+    ```
+    fin blt setup -D setup.strategy=install
+    ```
+
+### Existing project setup
+If you have an existing BLT project with Drupal install, you may prefer to configure your `local.settings.php`
+file and your `local.drush.yml` file. It is best practice to also update your blt.yml as outlined above, but
+it is not necessary to delete these files.
+
+1. configure your `docroot/sites/default/settings/local.settings.php` file with these variable values:
```

Comment:
Thanks. Yes, I wrote that blog post. I have those instructions for setting up new projects, but I wanted to include the instructions to modify the files in case the files already existed with additional data that would be lost if deleted.

Reply:
@shelane great, didn't saw that.